src/wagtail_ai/migrations/0002_populate_default_prompts.py

```diff
@@ -0,0 +1,31 @@
+# Created using Django 4.2.7 on 2023-12-06 10:07
+
+from django.db import migrations
+
+from wagtail_ai.models import DEFAULT_PROMPTS
+
+
+def set_default_ai_prompts(apps, schema_editor):
+    """
+    Loop through each default prompt and
+    populate the database.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+    for default_prompt in DEFAULT_PROMPTS:
+        Prompt.objects.update_or_create(
+            uuid=default_prompt["uuid"],
+            label=default_prompt["label"],
+            prompt=None,  # Left blank to allow users to override the prompt value and maintainers of wagtail AI to manage the prompt in the codebase.
+            description=default_prompt.get("description", ""),
+            method=default_prompt.get("method", None),
+        )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("wagtail_ai", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.RunPython(set_default_ai_prompts, migrations.RunPython.noop),
+    ]
```

Comment:
**Question:** Will we create a new migration file each time we update default prompts?

Reply:
One issue however, might be adding new default prompts, we'd need a new migration for that or maybe a management command for developers to run instead as this migration won't run again...

Reply:
Right, I see. It's good we have a way of identifying the default prompt by uuid then, haha.

Just to rant on unrelated topic:

I think I'd prefer to have a special purpose field for this such as `default_prompt_id` that's null on non-default fields so we're using specific fields for specific purposes, rather than making assumptions of what uuid field should be used for besides a random unique identifier. You rarely want to use audit/identification fields in novel ways for specific purposes as it makes it more complex to understand that if you are reading the model code. Could add a comment on the uuid field on the model if there is not one already about this use of the uuid field

I think it's fine to keep it as is, to be honest.

Reply:
I think your suggestion makes a lot more sense, and have refactored to support it, since it allows us to query default's prompts more efficently instead of having to iterate through `DEFAULT_PROMPTS` via `is_default_prompt()`.